elasticsearch/README.md

```diff
@@ -73,6 +73,12 @@ specific resources deployed:
   - One GCE VM for each data node
   - One Kubernetes Pod for each data VM
 
+The nodes are created on a Kubernetes cluster on a private cluster, meaning 
```

Comment:
The nodes are created on a Kubernetes cluster on a [private cluster](https://cloud.google.com/kubernetes-engine/docs/concepts/private-cluster-concept), meaning 
the nodes have reserved IP addresses only. This is useful so large clusters 
do not use up [external IP address quota](https://cloud.google.com/compute/quotas#ip_addresses).